include/librealsense2/dds/topics/notification/notification.idl

```diff
@@ -11,7 +11,7 @@ module librealsense
             {
                 module device
                 {
-                    struct notifications
+                    struct notification
```

Comment:
We changed the name of the topic? How come?

Reply:
I think that a topic name should indicate a data name for 1 instance, same as ROS topics.
`device-info`
`image`
`notification`

In a single message we have 1 notification so it should be different than the others